weatherapplication.client/src/app/core/pages/fivedays-weather-page/fivedays-weather-page.component.ts

```diff
@@ -1,10 +1,29 @@
-import { Component } from '@angular/core';
+import { Component} from '@angular/core';
+import {
+  CurrentWeatherQuery,
+  FiveDaysWeather,
+  HttpService,
+} from 'src/app/services/http.service';
 
 @Component({
-  selector: 'app-fivedays-weather-page',
+  selector: 'fivedays-weather-page',
```

Comment:
The `fivedays` word again.

Reply:
![image](https://github.com/Oleksandr-Ryabishchuk/WeatherApplication/assets/36698610/4d0a36a2-4911-436b-970c-fef8956ddec8)